src/ui/public/notify/notify.js

```diff
@@ -18,9 +17,10 @@ module.factory('Notifier', function () {
   return Notifier;
 });
 
-module.run(function ($timeout) {
-  // provide alternate methods for setting timeouts, which will properly trigger digest cycles
-  Notifier.setTimerFns($timeout, $timeout.cancel);
+module.run(function ($interval) {
+  // provide alternate methods for setting intervals
+  // which will properly trigger digest cycles
+  Notifier.setTimerFns($interval, $interval.cancel);
```

Comment:
Changing from `$timeout` to `$interval` seems like a strange move. If that is really the direction you want to go I suggest renaming the `setTimerFns` and `setTimer` and `clearTimer` function to imply that they are creating/clearing intervals.